tools/cal_scripts/FLNTUA/flntua_cal_parser.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# SPKIR calibration parser
+#
+# Create the necessary CI calibration ingest information from an SPKIR calibration file
+
+import csv
+import json
+import os
+import sys
+import datetime
+import re
+import copy
+from dateutil.parser import parse
+
+class FLNTUACalibration:
+    def __init__(self):
+        self.chl= None
+        self.vol = None
+        self.asset_tracking_number = None
+        self.serial = None
+        self.date = None
+        self.coefficients = {'CC_angular_resolution':1.096, 'CC_depolarization_ratio':0.039, 'CC_measurement_wavelength':700, 'CC_scattering_angle':140}
```

Comment:
PEP-8 - line too long

ProTip - use PyCharm (or another tool) that automatically highlights PEP-8 violations.